api/src/main/java/org/openmrs/module/pihapps/PihAppsUtils.java

```diff
@@ -0,0 +1,83 @@
+package org.openmrs.module.pihapps;
+
+import org.openmrs.Concept;
+import org.openmrs.ConceptName;
+import org.openmrs.api.ConceptNameType;
+import org.openmrs.api.context.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.Locale;
+
+@Component
+public class PihAppsUtils {
```

Comment:
This may have been a mistake to add this class, but these are things I moved over forem PihAppsConfig and LabOrderConfig that were more utility methods than retrievable configuration values